src/utils/registry.ts

```diff
@@ -303,6 +303,10 @@ export class Registry {
     return tokens ? path.join(browserDirectory, ...tokens) : undefined;
   }
 
+  exists(browserName: BrowserName): boolean {
```

Comment:
this is equal to the `shouldRetain` method below - let's just use it.

As a side note, `shouldRetain` is a pretty vague name - could you please rename it to something like `isSupportedBrowser(browserName)`?